src/test/java/com/exasol/adapter/dynamodb/queryrunner/DynamodbGetItemQueryPlanFactoryTest.java

```diff
@@ -0,0 +1,123 @@
+package com.exasol.adapter.dynamodb.queryrunner;
+
+import static com.exasol.adapter.dynamodb.queryrunner.TestSetup.*;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+
+import com.amazonaws.services.dynamodbv2.model.AttributeValue;
+import com.exasol.adapter.dynamodb.documentnode.dynamodb.DynamodbNodeVisitor;
+import com.exasol.adapter.dynamodb.documentnode.dynamodb.DynamodbString;
+import com.exasol.adapter.dynamodb.dynamodbmetadata.DynamodbKey;
+import com.exasol.adapter.dynamodb.dynamodbmetadata.DynamodbTableMetadata;
+import com.exasol.adapter.dynamodb.remotetablequery.*;
+
+public class DynamodbGetItemQueryPlanFactoryTest {
+    final TestSetup testSetup;
+
+    public DynamodbGetItemQueryPlanFactoryTest() throws IOException {
+        this.testSetup = new TestSetup();
+    }
+
+    @Test
+    void testSimplePrimaryKey() throws IOException, PlanDoesNotFitException {
+        final String filter = "test";
+        final ColumnLiteralComparisonPredicate<DynamodbNodeVisitor> selection = new ColumnLiteralComparisonPredicate<>(
+                ComparisonPredicate.Operator.EQUAL, COLUMN1_MAPPING, new DynamodbString(filter));
+        final RemoteTableQuery<DynamodbNodeVisitor> documentQuery = new RemoteTableQuery<>(TABLE_MAPPING,
+                TABLE_MAPPING.getColumns(), selection);
+
+        final DynamodbTableMetadata dynamodbTableMetadata = new DynamodbTableMetadata(
+                new DynamodbKey(COLUMN1_NAME, Optional.empty()), List.of(), List.of());
```

Comment:
Create a empty `Collections.emptyList()` variable and use in places of `List.of()`.